src/sentry/utils/snuba.py

```diff
@@ -210,7 +210,7 @@ def options_override(overrides):
 
 _snuba_pool = connection_from_url(
     settings.SENTRY_SNUBA,
-    retries=False,
+    retries=5,
```

Comment:
does this have exponential backoff or anything built in? 

just want to make sure a small blip doesn't lead to a thundering herd of retries.

Reply:
We can tune it to exactly how we want if that's a concern. I think by default it shouldn't retry based on status code. Only actual connection errors. Lemme check.

Reply:
We can can go crazy with this `Retry` object if we want to tune everything: https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#urllib3.util.retry.Retry